creator/data_reviews/models.py

```diff
@@ -0,0 +1,151 @@
+import uuid
+from django.db import models
+from django.contrib.auth import get_user_model
+from django_fsm import FSMField, transition
+
+from creator.fields import kf_id_generator
+from creator.studies.models import Study
+from creator.files.models import File
+
+User = get_user_model()
+
+
+def data_review_id():
+    return kf_id_generator("DR")
+
+
+class State(object):
+    NOT_STARTED = "not_started"
+    IN_REVIEW = "in_review"
+    WAITING = "awaiting_updates"
+    COMPLETED = "completed"
+    CLOSED = "closed"
+
+
+class DataReview(models.Model):
+    """
+    The iterative process between a data contributor and the Data Resource
+    Center where study data is continually supplied, validated, and refined
+    to meet quality standards.
+    """
+
+    class Meta:
+        permissions = [
+            ("list_all_datareview", "Show all data_reviews"),
+        ]
+
+    kf_id = models.CharField(
+        max_length=11,
+        primary_key=True,
+        default=data_review_id,
+        help_text="Kids First ID assigned to the data review",
+    )
+    uuid = models.UUIDField(
+        default=uuid.uuid4, help_text="UUID used internally"
+    )
+    created_at = models.DateTimeField(
+        auto_now_add=True,
+        null=True,
+        help_text="Time when the data_review was created",
+    )
+    creator = models.ForeignKey(
+        User,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="data_reviews",
+        help_text="The user who created the data review",
+    )
+    name = models.CharField(
+        max_length=256, help_text="Name of the data review")
+    description = models.TextField(
+        max_length=10000,
+        null=True,
+        blank=True,
+        help_text="Description of data review",
+    )
+    state = FSMField(
+        default="not_started", help_text="The current state of the data review"
```

Comment:
Do you want to use `STATE.NOT_STARTED` for the default?

Reply:
Ah oops yea